### PR TITLE
doc(featurizer): update for offline environments

### PR DESCRIPTION
**Pull request by parkererickson-tg**

### CODE REVIEW

File: pyTigerGraph/gds/gds.py [hunk 1]
```diff
@@ -1469,9 +1469,25 @@ def featurizer(
...
+        compatible with your database version.
+        Note: In environments not connected to the public internet, you can download the repository manually and use the featurizer
+        like this:
+        ```
+        import pyTigerGraph as tg
+        from pyTigerGraph.gds.featurizer import Featurizer
+
+        conn = tg.TigerGraphConnection(host="HOSTNAME_HERE", username="USERNAME_HERE", password="PASSWORD_HERE", graphname="GRAPHNAME_HERE")
+        conn.getToken(conn.createSecret())
+        feat = Featurizer(conn, repo="PATH/TO/MANUALLY_DOWNLOADED_REPOSITORY")
```
Comment (billshitg) [inline pyTigerGraph/gds/gds.py:1483]: why not using conn.gds.featurizer(repo = ...)?

Comment (parkererickson-tg) [reply pyTigerGraph/gds/gds.py:1483]: Do we want to add that parameter to the factory function? I was thinking it makes the factory a bit more clunky. Can definitely add it though, there wasn't a strong reason why I didn't add it in.

Comment (billshitg) [reply pyTigerGraph/gds/gds.py:1483]: looks like it is already in the factory function lol so no need for extra work

**Review by billshitg — APPROVED** [review]

**Review by qe-tigergraph — COMMENTED** [review]
Unit Test: SUCCESS, e2e Test: SUCCESS, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/742/

**Review by qe-tigergraph — APPROVED** [review]
QE Approved

**Pull request merged**